src/Modules/CalcDefence.lua

```diff
@@ -2393,13 +2396,18 @@ function calcs.defence(env, actor)
 					})
 				end
 			end
+
+			local takenMult = output[damageType.."TotalHitPool"] / output[damageType.."MaximumHitTaken"] / enemyDamageMult
+
 			t_insert(breakdown[damageType.."MaximumHitTaken"], s_format("Total Pool: %d", output[damageType.."TotalHitPool"]))
-			t_insert(breakdown[damageType.."MaximumHitTaken"], s_format("Taken Mult: %.3f",  output[damageType.."TotalHitPool"] / output[damageType.."MaximumHitTaken"]))
+			t_insert(breakdown[damageType.."MaximumHitTaken"], s_format("/ %.2f (modifiers to damage taken)", takenMult))
+			t_insert(breakdown[damageType.."MaximumHitTaken"], s_format("/ %.2f (modifiers to enemy damage)", enemyDamageMult))
+
 			if useConversionSmoothing then
-				t_insert(breakdown[damageType.."MaximumHitTaken"], s_format("Approximate maximum hit you can take: %d", output[damageType.."MaximumHitTaken"]))
+				t_insert(breakdown[damageType.."MaximumHitTaken"], s_format("= %d (approximate)", output[damageType.."MaximumHitTaken"]))
 				t_insert(breakdown[damageType.."MaximumHitTaken"], s_format("You would take %d damage from such a hit.", takenHitFromDamage(output[damageType.."MaximumHitTaken"])))
```

Comment:
Its accounted for on line 2369 and 2371 and then used here

Reply:
No this is reapplying the enemyDamageMult to the damage taken by the simulation is correct (it doesn't factory this already as it is used to calculate it without). The suggested change would be needed I am pretty sure.

Reply:
Oh i see, ok updated

Reply:
Oh also for the other place where its not used (for the damage conversion table), that was done intentionally as its only applied at the end @Edvinas-Smita .

Reply:
Applying the multiplier on line 2388 is required for the damage breakdown to show the correct "Taken" value when armour is involved.

https://pobb.in/w-Copz_QPgiT
![image](https://user-images.githubusercontent.com/36479307/209474781-4d62eef1-a401-431f-8544-2ff0ccf5c0ba.png)

<br/>
<br/>

P.S. I just noticed that I left the keys on lines 2393-2395 messed up. It should be
```
incoming = s_format("%d", convertedDamage),
multi = s_format("x%.3f", multi),
taken = s_format("%.0f", convertedDamage * multi),
```
instead of
```
initial = s_format("%d", convertedDamage),
taken = s_format("x%.3f", multi),
final = s_format("%.0f", convertedDamage * multi),
```
\- Probably small enough of a change to sneak it into this PR.

Reply:
Yea that was done intentionally. I can change it but @QuickStick123 wanted it like that. And for the other part, its unrelated to this PR so that can be done in different one.

Reply:
You probably do want to apply the multi there on second thought as to get the correct final multi from the computed numbers adding final damages and dividing you would need to consider this. I would update it.

Reply:
Alright updated